cli/command/service/list.go

```diff
@@ -40,6 +36,8 @@ func newListCommand(dockerCli *command.DockerCli) *cobra.Command {
 
 	flags := cmd.Flags()
 	flags.BoolVarP(&opts.quiet, "quiet", "q", false, "Only display IDs")
+	flags.BoolVar(&opts.noTrunc, "no-trunc", false, "Do not truncate the output")
```

Comment:
@yongtang perhaps, yes. My initial thought was "do we need a `--no-trunc` flag? That's when I started thinking if we perhaps need other options for the format.

It's just a thought though, so perhaps @vdemeester or @dnephin has some thoughts